tests/query/processors/test_filter_in_select_optimizer.py

```diff
@@ -50,107 +59,241 @@ def subscriptable_reference(name: str, key: str) -> SubscriptableReference:
     )
 
 
-mql_test_cases: list[tuple[str, dict]] = [
-    (
-        "sum(`d:transactions/duration@millisecond`){status_code:200} / sum(`d:transactions/duration@millisecond`)",
-        {
-            Column("_snuba_metric_id", None, "metric_id"): {
-                Literal(None, 123456),
-            }
-        },
-    ),
-    (
-        "sum(`d:transactions/duration@millisecond`){status_code:200} / sum(`d:transactions/duration@second`)",
-        {
-            Column("_snuba_metric_id", None, "metric_id"): {
+def equals(lhs: Expression, rhs: Expression) -> FunctionCall:
```

Comment:
There's a file called `dsl.py` that has a bunch of helper functions like this, I would add these there so other people can use them in the future.